app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Posts < ApplicationRecord
+  belongs_to :user, foreign_key: :author
```

Comment:
[REQUIRED] This should be `belong_to :author, className: 'User'` instead. Kindly fix this so that your code can work as expected.